src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -36,47 +36,36 @@
 import org.zaproxy.zap.extension.pscan.PassiveScanThread;
 import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
 
+/**
+ * Plugin that scans the content for private IP V4 addresses as well as 
+ * Amazon EC2 private hostnames (for example, ip-10-34-56-78).
+ */
 public class TestInfoPrivateAddressDisclosure extends PluginPassiveScanner {
 
-	/**
-	 * Prefix for internationalised messages used by this rule
-	 */
-	private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
-	
-    private static final String REGULAR_IP_OCTET = "\\b(25[0-5]|2[0-4][0-9]|1?[0-9]{1,2})";
-    private static final String REGULAR_PORTS = "\\b(6553[0-5]|65[0-5][0-2][0-9]|6[0-4][0-9]{4}|[0-5]?[0-9]{0,4})";
-    
-    // Private IP's including localhost
+    /**
+     * Prefix for internationalised messages used by this rule
+     */
+    private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
+
+    private static final String REGULAR_IP_OCTET = "\\b(25[0-5]|2[0-4][0-9]|1?[0-9]{1,2})\\b";

```

Comment:
For the record there are couple of word boundaries that could be dropped (e.g. drop from the octet and add to the begin/end of the addresses; drop from the beginning of the port) and some unnecessary escaped characters.